user_docs/en/userGuide.t2t

```diff
@@ -1850,6 +1850,14 @@ The combo box has the following options:
 - No: Don't use UIA, even if NVDA is unable to inject in process. This may be useful for developers debugging issues with IA2 and want to ensure that NVDA does not fall back to UIA.
 -
 
+==== Use UI automation to access Microsoft  Excel spreadsheet controls when available ====[AdvancedSettingsUseUiaForExcel]

```

Comment:
In https://github.com/nvaccess/nvda/pull/12034#issuecomment-773847508, @feerrenrut recommanded not to include the category name in the tag, especially for advanced settings that may migrate in another category one day.

```suggestion
==== Use UI automation to access Microsoft  Excel spreadsheet controls when available ====[UseUiaForExcel]

```